src/store/types.ts

```diff
@@ -3,3 +3,9 @@ export interface LoadableEntityState<T> {
   error?: string;
   isLoading: boolean;
 }
+
+export interface SendInterface {
```

Comment:
Does this interface correspond to shared `src/store/types.ts`?  
Seems like a particular case